src/main/java/Chris.java

```diff
@@ -0,0 +1,116 @@
+import java.util.Scanner;
+
+import task.Deadlines;
+import task.Events;
+import task.TaskLists;
+import task.Tasks;
+import task.ToDos;
+import exceptions.InputException;
+public class Chris {
+    public static void main(String[] args) throws InputException {
+        String logo = "      ___           ___           ___                       ___     \n" +
```

Comment:
logo should be a constant Final type variable as the value of the logo does not change through the course of this code.